x-pack/legacy/plugins/infra/server/lib/adapters/framework/adapter_types.ts

```diff
@@ -76,6 +76,7 @@ export interface InfraWrappableRequest<Payload = any, Params = any, Query = any>
   payload: Payload;
   params: Params;
   query: Query;
+  [internalInfraFrameworkRequest]: any;
```

Comment:
What is this change?

Reply:
The type checker was getting upset that `req[[internalInfraFrameworkRequest]` wasn't specified here, although according to the CI it doesn't seem completely satisfied with that either.